### DISCUSSION TITLE
user to drop a custom font in

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#413] [Feature Requests] [answered]
I know we can select an image from the sidebar. Would be great if a user to could select a font from their computer.

**Comment by steam0r** [comment, marked as answer]
this should work, right?

![Bildschirmfoto 2022-11-28 um 11 11 59](https://user-images.githubusercontent.com/151271/204251634-b50623d4-0ffc-4804-82a5-0e759edf24b8.png)

**Reply by marioepsley** [reply]
ooo I'll try this
